docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Participating in the Athens Community"
+date: 2018-02-11T15:57:56-05:00
+---
+
+Absolutely everyone is welcome to join our community at any time! We
+are a friendly and inclusive group and we'd love to have you. We have three roles in the Athens community:
+
+- Community member
+- Contributor
+- Maintainer (we also call this core maintainer sometimes. They're the same thing)
+
+Read on to find out more!
+
+# Community Members
+
+Community members are folks who decide they want to get involved with our
+community. Absolutely anyone can do that whenever they want. If you want
+to get involved, that doesn't mean you have to continue being involved, but
+we hope our community is welcoming and the work is interesting enough to 
+convince you to say :)
+
+We'll provide all the support we can possibly provide to help you contribute
+in any way you'd like. If you're considering joining us, here are some ideas
+for how you can get involved:
+
+- Comment on an [issue](https://github.com/gomods/athens/issues) that you're
+interested in
+- Submit a [pull request](https://github.com/gomods/athens/pulls) (PR) to fix 
+an issue, or to improve something that doesn't have an issue
+- Review a PR that you're interested in
+- Join us at a weekly [development meeting](https://docs.google.com/document/d/1xpvgmR1Fq4iy1j975Tb4H_XjeXUQUOAvn0FximUzvIk/edit#)
+(or more than one!)
+- Come chat with us in the [gophers slack](https://invite.slack.golangbridge.org/) in the `#athens` channel
+- ... and anything else that's appropriate for you!
+
+# Contributors
+
+As you participate in the community more and more, you'll have the opportunity 
+to become a contributor. Here's what being a contributor means, and what you
+should do to become one.
+
+## What Being a Contributor Means
+
+As a contributor, you'll have issues assigned to you and you'll be requested 
+to review pull requests (PRs) via the 
+[Github pull request review system](https://help.github.com/articles/about-pull-request-reviews/). 
+We rely heavily on Github PR reviews, so if you review a PR as a contributor, you can decide 
+when that PR is ready to be merged.
```

Comment:
So a contributor is a member of the gomods org (so that issues can be assigned to them) and is also has write permissions on the repo?

Reply:
Contributors have read access to the repository (see screenshot), so they can be assigned to issues and PRs. I'll call out here specifically that they have read access to the repository